round4/team-07.md

```diff
@@ -0,0 +1,142 @@
+
+16. As a user, I want the currency converter Chrome extension to offer both dark and light mode themes, allowing me to switch between them based on my preferences and reducing eye strain during late-night browsing sessions.
+
+## Notes - AT
+
+Since many variations of this Chrome extension already exist, it may be challenging to distinguish our product from others. Since this is a relatively simple product, I think that a potentially unique approach for us could be to make the UI especially compelling and user friendly. We are considering an option where the conversion happens automatically when one highlights a number on the webpage. An even further step would be to do an automatic conversion, the way certain translation Chrome extensions work.
+
+## References & Inspiration - AT
+
+We were inspired by the Currency Converter Widget Chrome extension which already exists, as a guideline of the general features which we want to include for our own Chrome extension. While we are not aiming to replicate this fully, we want to make sure we include the features that already exist in this extension and build upon them for our own project. The extension can be found here: https://chromewebstore.google.com/detail/currency-converter-widget/bnpalipgomknhgbmgelaplknnmckljaf
+
+This extension will also be used to guide our UI improvements, since we will identify pain points in existing extensions and use them to improve our own extension. 
+
+
+
+## Technical Details
+
+### User Interface - RT
+
+When users interact with the extension, they will typically see an icon in the Chrome toolbar, representing the currency converter. Upon clicking the icon, a small pop-up window will appear. This window will include:
+- Input fields for the source currency amount and the selection of the source currency.
+- Dropdown menus or search bars for selecting the target currency.
+- Real-time conversion results displayed prominently, showing the converted amount.
+- Additional settings or preferences accessible through a gear icon, allowing users to customize the extension behavior (such as language preferences, decimal precision, etc.).
+- Buttons or shortcuts for quick actions, such as swapping source and target currencies, clearing input fields, accessing historical exchange rate data, and a bookmark for previous conversions saved by the user.
+  
+Users will interact with the extension primarily by inputting the source currency amount and selecting the source and target currencies using dropdown menus or search bars. The conversion results will be updated dynamically as users make changes, providing instant conversions. Additionally, users can customize their experience by adjusting settings or toggling between dark and light mode themes to suit their preferences.
+
```

Comment:
New version:
- No input fields for amounts: Dropdown to choose currencies
- Instead of providing amounts, will have user select currency amounts to convert
- Once amount is selected, several options: For instance have hover box with converted amount
- The selection can be difficult (see Amazon example), so having an input field for amounts could also be a backup solution — in addition to the selection @akankshatripathy 
- Possibly display the converted result using a hoverbox [optional goal]

MVP:
1. Maybe first build the popup with input field (check you are doing backend, fetching the currency conversion rates, etc.)
2. Add the selection functionality, where a user can also select an amount on the page
3. With selection, how is the currency displayed
    - console.log(...)
    - window.alert(...)
    - the next best version could be the hoverbox

Second feature (if time):
1. Extract all the dollar amounts in the page
2. Sort them
3. And create a "scale in the target currency" that uses this data